.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/Source/JavaScriptCore @WebKit/jsc-reviewers
```

Comment:
It is showing a warning that "this CODEOWNERS file contains errors" on the "Files changed" page 😕.

Maybe it has to do with "Users must have read access to the repository and teams must have explicit write access, even if the team's members already have access."

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
Yes, currently, it is pending state. I think @JonWBedard has an ability to change it.

Reply:
Maybe we will need to list the reviewers individually?

Reply:
Once jsc-reviewers gets write access as reviewers have, then we can just specify jsc-reviewers as a review request target.